--- Mercurial.ini.orig	Fri Jun  6 09:33:30 2008
+++ Mercurial.ini	Fri Jun  6 09:50:44 2008
@@ -2,11 +2,8 @@
 ; per-user basis, please edit the following file instead, where
 ; USERNAME is your Windows user name:
 ;   C:\Documents and Settings\USERNAME\Mercurial.ini
 
-[ui]
-editor = notepad
-
 [extensions]
 ; The win32text extension is available and installed by default.  It
 ; provides built-in Python hooks to perform line ending conversions.
 ; This is normally much faster than running an external program.
@@ -83,4 +80,14 @@
 ecmerge.gui=True
 
 filemerge.args=-left $other -right $local -ancestor $base -merge $output
 filemerge.gui=True
+
+[diff]
+git = 1
+showfunc = 1
+
+[extensions]
+hgext.graphlog =
+
+[defaults]
+diff =-p -U 8
